schema/features/autofill.ts

```diff
@@ -0,0 +1,22 @@
+import { Feature, SubFeature } from '../feature';
+
+// Type of the feature `settings` object
+type SettingsType = undefined;
+
+// Any subfeatures that have typed `settings` should be defined here.
+// Subfeatures without settings (or just string:string mappings for settings) will be automatically validated.
+type SubFeatures<VersionType> = {
+    canImportFromGooglePasswordManager?: SubFeature<
+        VersionType,
+        {
+            launchUrl: string;
+            javascriptConfig: string;
```

Comment:
We still will want to change this once the feature gets used, fine to merge as is though.

Reply:
Yeah, currently waiting for @CDRussell to confirm what this schema should be.

Reply:
Feel free to merge this without waiting on me, and if we need to update this again it will be done as part of https://github.com/duckduckgo/privacy-configuration/pull/2453

Reply:
I think it will be probably:
```js
            javascriptConfig: CSSInjectFeatureSettings<ImportFromGooglePasswordManager>;
```

Where:
```ts
type ButtonConfig = {
  shouldAutotap: boolean;
  path: string;
  selectors: string[];
  labelTexts: string[];
};

type ImportFromGooglePasswordManager = {
  settingsButton: ButtonConfig;
  exportButton: ButtonConfig;
  signInButton: ButtonConfig;
};
```

Assuming the config looks like:
```json
"javascriptConfig": {
  "domains": [
    {
      "domain": [
        "google.com",
        "localhost"
      ],
      "patchSettings": [
        {
          "path": "",
          "op": "add",
          "value": {
            "settingsButton": {
              "shouldAutotap": false,
              "path": "/",
              "selectors": [
                "a[href*='options']"
              ],
              "labelTexts": [
                "Password options"
              ]
            },
            "exportButton": {
              "shouldAutotap": false,
              "path": "/options",
              "selectors": [
                "c-wiz[data-node-index*='2;0'][data-p*='options']",
                "c-wiz[data-p*='options'][jsdata='deferred-i4']"
              ],
              "labelTexts": [
                "Export"
              ]
            },
            "signInButton": {
              "shouldAutotap": false,
              "path": "/intro",
              "selectors": [
                "a[href*='ServiceLogin'][target='_top']"
              ],
              "labelTexts": [
                "Sign in"
              ]
            }
          }
        }
      ]
    }
  ]
}

```

Reply:
Looks good to me